doc/utils/sui-setup.sh

```diff
@@ -3,7 +3,12 @@
 #!/bin/bash
 # Prereqs: Rust Cargo, Git CLI, and GitHub account
 # Usage: set up environment for Sui development
-# Run `sui-setup.sh` in the directory to download source
+# First run these commands tow download the script
+# curl https://raw.githubusercontent.com/MystenLabs/sui/main/doc/utils/sui-setup.sh -o sui-setup.sh
+# chmod 755 sui-setup.sh
```

Comment:
If someone is reading these lines, shouldn't they have gone through this already?

Reply:
I am hoping to link to this file and yes reiterate these steps pre-download. But until that happens, we need this documented somewhere.

I haven't gotten any sort of formal OK to recommend this from our docs... WDYT?